GREEDY/OCRs/TesseractOcr.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Drawing;
+using GREEDY.Extensions;
+using GREEDY.Models;
+using Tesseract;
+
+namespace GREEDY.OCRs
+{
+    public class TesseractOcr : IOcr
+    {
+        private readonly TesseractEngine _tesseract;
+
+        public TesseractOcr()
+        {
+            _tesseract = new TesseractEngine
```

Comment:
I believe this will crash since Tesseract does not support Lithuanian language and TesseractAndCube mode. Also it didn't seem to read receipts correctly in other modes

Reply:
TesseractOcr was not using in our program, so I did not pay attention this time. Yes, possibly this will crash, but to implement Lithuanian language to TesseractOcr will be future our task. :) I tried only change a coding style and add few features as shop method and others, but not to fix all code